scripts/common.js

```diff
@@ -228,13 +198,20 @@ export function itemTransformer(item) {
  * adds it to the specified container.
  * @param {object} hit The query hit with the card data.
  * @param {element} $container The container element to add the card to
- * @param {element} $template The template element to use (optional)
  * @returns {element} The card element
  */
-export function addCard(hit, $container, $template) {
-  if (!$template) $template = document.getElementById('post-card');
-  const $item = $template.content.cloneNode(true).firstElementChild;
-  fillData($item, hit);
+export function addCard(hit, $container) {
+  const $item = createTag('div', {'class': 'card'});
+  $item.innerHTML = `
+  <div class="hero">
+    <a href="/${hit.path}" title="${hit.title}"><img class="lazyload" src="#" data-src="${hit.hero}" alt="${hit.title}"></a>
```

Comment:
Might be minor and was the case before already but don't you get 2 slashes for the `href` ?

Reply:
No :) the `path` property does not start with a slash